dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstance.java

```diff
@@ -79,6 +80,8 @@
 
     private String coordinates;
 
+    private Date lastSynced = new Date( 0 );

```

Comment:
Perhaps rename to "lastSynchronized".

A bit more formal and consistent as i think we use this other places.